### PR TITLE
docs: Update installation guide for Ubuntu.

### LINKED ISSUE #214: Update installation guide for ubuntu
Include in the document the support for ubuntu 18.04

### PR DESCRIPTION
**Pull request by GabyCT**
Include in the document the support for ubuntu 18.04.

Fixes #214

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: install/ubuntu-installation-guide.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 > **Notes:**
 >
 > - Kata Containers packages are available for [Ubuntu\*](https://www.ubuntu.com)
->   versions **16.04** and **17.10** (currently `x86_64` only).
+>   versions **16.04**, **17.10** and **18.04** (currently `x86_64` only).
```
Comment (amshinde) [inline install/ubuntu-installation-guide.md]: @GabyCT Your commit message says 18.10, but you have added 18.04 here.

Comment (GabyCT) [reply install/ubuntu-installation-guide.md]: @amshinde sorry my mistake, it is 18.04

**Comment by jodh-intel** [comment]
lgtm

**Comment by caoruidong** [comment]
lgtm

**Pull request merged**